gadgets/trace_tcp/test/trace_tcp_test.go

```diff
@@ -139,6 +175,11 @@ func TestTraceTCP(t *testing.T) {
 				utils.NormalizeString(&e.Timestamp)
 				utils.NormalizeInt(&e.Pid)
 				utils.NormalizeInt(&e.NetNsID)
+				// Checking the ports is a little bit complicated as successive
+				// calls to curl with --local-port fail because of
+				// https://github.com/curl/curl/issues/6288
```

Comment:
:(

Reply:
I've been thinking about this. Perhaps we should be using something else and not existing binaries/images for our tests. One idea is to use python to generate the very specific events we need.